frontend/www/js/omegaup/components/user/Dependents.vue

```diff
@@ -25,11 +37,46 @@
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
+import * as ui from '../../ui';
 import { types } from '../../api_types';
 
 @Component
 export default class UserDependents extends Vue {
   @Prop() dependents!: types.UserDependentsPayload[];
+  @Prop() userVerificationDeadline!: Date | null;
+
   T = T;
+  ui = ui;
+
+  get bannerColor() {
+    const today = new Date();
+    const deadline = new Date(this.userVerificationDeadline as Date);
+    if (deadline.toDateString() === today.toDateString()) {
+      return 'bg-danger';
+    } else {
+      return 'bg-warning';
+    }
+  }
```

Comment:
```suggestion
  get bannerColor() {
    const today = new Date();
    const deadline = new Date(this.userVerificationDeadline as Date);
    if (deadline.toDateString() === today.toDateString()) {
      return 'bg-danger';
    }
    return 'bg-warning';
  }
```
No olvides el [guard clause pattern](https://refactoring.guru/es/replace-nested-conditional-with-guard-clauses)

Reply:
Esta bien